modules/kmm-customizing-upgrades-for-kernel-modules.adoc

```diff
@@ -37,15 +36,30 @@ The update should be atomic; that is, both the `containerImage` and `version` fi
 
 . Terminate any workload using the kernel module on the node being upgraded.
 
-. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node. This unloads the kernel module from the node.
+. Remove the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label on the node.
+This unloads the kernel module from the node:
++
+[source,terminal]
+----
+$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>-
+----
 
-. If required, perform any additional maintenance required on the node for the kernel module upgrade.
+. If required, as the cluster administrator, perform any additional maintenance required on the node for the kernel module upgrade.
++
+If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
 
 . Add the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>=$moduleVersion` label to the node. `$moduleVersion` should be equal to the new value of the `version` field in the `Module` CR.
-
-. Restore any workload that leverages the kernel module on the node.
-
++
+[source,terminal]
+----
+$ oc label node/<node_name> kmm.node.kubernetes.io/version-module.<module_namespace>.<module_name>=<desired_version>
+----
++
 [NOTE]
 ====
-If no additional upgrading is needed, you can skip Steps 3 through 6 by updating the `kmm.node.kubernetes.io/version-module.<module-namespace>.<module-name>` label value to the new `$moduleVersion` as set in the `Module`.
+Due to Kubernetes limitations in label names, the combined length of `Module` name and namespace must not exceed 39 characters.
```

Comment:
```suggestion
Because of Kubernetes limitations in label names, the combined length of `Module` name and namespace must not exceed 39 characters.
```
I wonder if "because of" might translate better.

Reply:
Made this change.